src/components/App.js

```diff
@@ -114,6 +116,9 @@ class App extends Component {
           closeModal={this.closeModal}
           modelToggle={this.state.modalIsOpen}
           data={this.state}
+          handleDelete={this.handleDelete}
+          handleSelected={this.handleSelected}
```

Comment:
Can we rename the handler to `handleDelete` to `onRemoveTodo` and can you tell me what is the purpose of `handleSelected`